src/config.ts

```diff
@@ -1222,4 +1231,24 @@ export default {
    * there can be in that history.
    */
   BUFFERED_HISTORY_MAXIMUM_ENTRIES: 200,
+
+  /**
+   * Minimum buffer (in seconds ) we should have, regardless of memory
```

Comment:
Very minor, but there are spaces at multiple unexpected place. Here before the closing parenthesis and multiple times in that PR, before a comma